package.json

```diff
@@ -264,7 +278,7 @@
     "minimist": "^1.2.3",
     "mv": "^2.1.1",
     "node-polyfill-webpack-plugin": "^1.1.0",
-    "pi-ui": "https://github.com/decred/pi-ui",
+    "pi-ui": "https://github.com/bgptr/pi-ui#improve-textinput",
```

Comment:
Upstream PR was merged, so ready to change back.

Reply:
@matheusd Thanks for the review, good catches.
The error in the sent tab was caused unfortunately by the pi-ui TextInput component. It was my bad, opened a new PR with the fix. Until it will be merged, I updated the package.json with the new fixing branch name.
The "Prop id is marked as required..." message was a warning. I fixed it and the similar warnings too.
The font size of placeholders is slightly higher indeed (16px) and it was dictated by the design.

Reply:
@matheusd I examined the placeholders again and you are right, something was weird about them. I figured out that the font family was inherited from `pi-ui`-s `TextInput` component and I missed overwriting it. 
I fixed them and a few misalignments that I found with fresh eyes.
Thanks.